example/spec/syncGranule/SyncGranuleSuccessSpec.js

```diff
@@ -0,0 +1,78 @@
+const fs = require('fs');
+const path = require('path');
+const { s3, s3ObjectExists } = require('@cumulus/common/aws');
+const { buildAndExecuteWorkflow, LambdaStep } = require('@cumulus/integration-tests');
+
+const { loadConfig, templateFile } = require('../helpers/testUtils');
+const config = loadConfig();
+const lambdaStep = new LambdaStep();
+const taskName = 'SyncGranule';
+
+const outputPayloadTemplateFilename = './spec/syncGranule/SyncGranule.output.payload.template.json'; // eslint-disable-line max-len
+const templatedOutputPayloadFilename = templateFile({
+  inputTemplateFilename: outputPayloadTemplateFilename,
+  config: config.SyncGranule
+});
+const expectedPayload = JSON.parse(fs.readFileSync(templatedOutputPayloadFilename));
```

Comment:
Not necessary to fix here, but in general, it's really hard to track down what's actually being asserted when it's put in separate files and everything is just variable comparison.